src/doc_types/hosts.ts

```diff
@@ -95,6 +98,7 @@ export function initializeHosts(typeDef: TypeDef, { logger }: SlingshotContext)
         min: 0,
         max: Number.MAX_SAFE_INTEGER,
       });
+       const diskIoTime = getMetric(now, 'diskIoTime', typeDef, {min: 0, max: Number.MAX_SAFE_INTEGER})
```

Comment:
This is probably not a big deal for this purpose, but `system.diskio.io.time` looks like it's a counter, so this modeling wouldn't work well if we wanted to do graphs, I think.

Reply:
Here's a link (internal) if you want to take a look at some of the values from real hosts https://overview.elastic-cloud.com/app/dashboards#/view/d6a72460-3e2a-11ed-b9cc-9db10f1626d2

Reply:
Thanks @matschaffer.  Doing further investigation but will commit for now.  I am thinking this is a gauge if it's the same field as ECS Host disk fields: https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-disk-read-bytes